src/test/java/com/wikia/webdriver/testcases/mercurytests/recentwikiactivity/RecentWikiActivityTest.java

```diff
@@ -0,0 +1,74 @@
+package com.wikia.webdriver.testcases.mercurytests.recentwikiactivity;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.RecentWikiActivity;
+import com.wikia.webdriver.elements.mercury.pages.MainPage;
+import com.wikia.webdriver.elements.mercury.pages.RecentWikiActivityPage;
+
+import org.testng.annotations.Test;
+
+@Execute(onWikia = "aga")
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+public class RecentWikiActivityTest extends NewTestTemplate {
+
+  @Test(groups = "mercury_recentWikiActivity_anonNavigateToRWAfromMenu")
+  public void mercury_recentWikiActivity_anonNavigateToRWAfromMenu() {
+    new MainPage()
+        .open()
+        .getTopBar()
+        .openNavigation()
+        .openRecentWikiActivity();
+  }
+
+  @Test(groups = "mercury_recentWikiActivity_anonOpenDiffPageAndGoBack")
+  public void mercury_recentWikiActivity_anonOpenDiffPageAndGoBack() {
+    this.openDiffPageAndGoBack();
+  }
+
+  @Test(groups = "mercury_recentWikiActivity_blockedUserOpenDiffPageAndGoBack")
+  @Execute(asUser = User.CONSTANTLY_BLOCKED_USER)
+  public void mercury_recentWikiActivity_blockedUserOpenDiffPageAndGoBack() {
+    this.openDiffPageAndGoBack();
+  }
+
+  @Test(groups = "mercury_recentWikiActivity_loggedUserOpenDiffPageAndSubmitWithoutSummary")
+  @Execute(asUser = User.USER)
+  public void mercury_recentWikiActivity_loggedUserOpenDiffPageAndSubmitWithoutSummary() {
+    this.openDiffPage()
+        .submitWithoutSummary()
+        .displaySuccessNotification();
+  }
+
+  @Test(groups = "mercury_recentWikiActivity_loggedUserOpenDiffPageAndSubmitWithSummary")
+  @Execute(asUser = User.USER)
+  public void mercury_recentWikiActivity_loggedUserOpenDiffPageAndSubmitWithSummary() {
+    this.openDiffPage()
+        .submitWithSummary()
+        .displaySuccessNotification();
+  }
+
+  @Test(groups = "mercury_recentWikiActivity_loggedUserOpenDiffPageAndGoBackFromSummary")
+  @Execute(asUser = User.USER)
+  public void mercury_recentWikiActivity_loggedUserOpenDiffPageAndGoBackFromSummary() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)